caluma/caluma_form/tests/test_answer.py

```diff
@@ -256,6 +256,35 @@ def test_save_default_answer_python_api(
             api.save_default_answer(question, user=admin_user, value=inp["value"])
 
 
+@pytest.mark.parametrize("question__type,answer__value", [(Question.TYPE_FLOAT, 0.1)])
+def test_save_calculated_dependency_default_answer(
+    db,
+    snapshot,
+    question,
+    answer,
+    question_option,
+    document_factory,
+    answer_factory,
+    answer_document_factory,
+    question_factory,
```

Comment:
There are a couple of unused fixture / factories used here.

Reply:
removed